blablas/blablas.yaml

```diff
@@ -0,0 +1,119 @@
+- date: 2019-03-21
```

Comment:
I'm not sure if this is the best approach to handle data because we are planned to add Notes to episode markdown file: see #5

Reply:
oh I see